modules/app/components/layout/header/NetworkAlertModal.tsx

```diff
@@ -73,6 +81,8 @@ const NetworkAlertModal = ({
               Your wallet is connected to an unsupported network, please switch it to{' '}
               {SupportedNetworks.MAINNET} or {SupportedNetworks.GOERLI} to continue.
             </Text>
+            {/* <Button onClick={() => handleSwitchNetwork(SupportedNetworks.MAINNET)}>Switch to mainnet</Button>
```

Comment:
nice, kinda happy to get rid of this. Also, just flagging the `network mismatch` error above, probably need to adjust the code there too.